geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -1867,6 +1877,12 @@ void shutdownMessageReceived(InternalDistributedMember theId, String reason) {
     handleManagerDeparture(theId, false, reason);
   }
 
+  long getNonLocatorViewMembers() {
+    return getViewMembers().stream()
```

Comment:
I think instead of `getViewMembers()`, it is better to use `distribution.getMembersNotShuttingDown()` to exclude the members that are shutting down. Please see `getOtherDistributionManagerIds()` and `getOtherNormalDistributionManagerIds()`.

Reply:
thanks for pointing out those methods. updated the PR to use that instead